src/chart/cartesian.js

```diff
@@ -66,31 +66,31 @@ export default function(xScale, yScale) {
 
             var svg = containerDataJoin(container, [data]);
             svg.enter().html(
-                '<g class="title"> \
-                    <g layout-style="height: 0; width: 0"> \
-                        <text class="label"/> \
+                '<g class="plot-area-container"> \
+                    <rect class="background" \
+                        layout-style="position: absolute; top: 0; bottom: 0; left: 0; right: 0"/> \
+                    <g class="axes-container" \
+                        layout-style="position: absolute; top: 0; bottom: 0; left: 0; right: 0"> \
+                        <g class="x-axis" layout-style="height: 0; width: 0"/> \
+                        <g class="y-axis" layout-style="height: 0; width: 0"/> \
                     </g> \
+                    <svg class="plot-area" \
+                        layout-style="position: absolute; top: 0; bottom: 0; left: 0; right: 0"/> \
                 </g> \
-                <g class="y-axis"> \
+                <g class="x-axis label"> \
```

Comment:
I have previously used `.label` to style the text labels, it's not a big deal if these styles are also applied to the `g`, and I could always update to `text.label`, but maybe `.label-container` would make sense here?